<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back_to_main">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rate_of_payment">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_tax">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Капиталлизация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма прибыли:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_of_payment">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>180</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_time">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_amount_profit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rate_of_exchange">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>9.8</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time_replament">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_r">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate_deposit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Получить результат</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_amount_money">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>230</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_money">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_percent">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Частичные снятия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pop_num">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawal">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pop_time">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="withdrawal_time">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_pop">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string> Добавить пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_with">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string> Добавить снятие</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
